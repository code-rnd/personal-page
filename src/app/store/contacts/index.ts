export { contactsActions } from "./contacts.actions";
export { contactsReducer } from "./contacts.reducer";
export * from "./contacts.selector";
