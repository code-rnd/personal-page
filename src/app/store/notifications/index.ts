export { notificationsActions } from "./notifications.actions";
export { notificationsReducer } from "./notifications.reducer";
export * from "./notifications.selector";
