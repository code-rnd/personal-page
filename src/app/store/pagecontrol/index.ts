export { pageControlActions } from "./pagecontrol.actions";
export { pagecontrolReducer } from "./pagecontrol.reducer";
export * from "./pagecontrol.selector";
