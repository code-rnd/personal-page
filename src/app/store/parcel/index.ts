export { parcelActions } from "./parcel.actions";
export { parcelReducer } from "./parcel.reducer";
export * from "./parcel.selector";
