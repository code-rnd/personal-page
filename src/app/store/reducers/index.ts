import { combineReducers } from "redux";

import { historyReducer as history } from "../history";
import { authorizationReducer as authorization } from "../authorization";
import { routeReducer as route } from "../route";
import { pagecontrolReducer as pagecontrol } from "../pagecontrol";
import { contactsReducer as contacts } from "../contacts";
import { notificationsReducer as notifications } from "../notifications";

export const reducers = combineReducers({
  history,
  authorization,
  route,
  pagecontrol,
  contacts,
  notifications,
});
