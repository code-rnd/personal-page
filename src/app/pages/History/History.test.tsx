import React from "react";
import renderer from "react-test-renderer";
import { Provider } from "react-redux";

import { History } from "./index";
import { store } from "../../store";

it(`Компонент ${History.name}, отрисован корректно`, () => {
  const Page = renderer
    .create(
      <Provider store={store}>
        <History />
      </Provider>
    )
    .toJSON();

  expect(Page).toMatchSnapshot();
});
