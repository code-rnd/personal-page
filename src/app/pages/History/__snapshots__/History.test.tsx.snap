// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Компонент undefined, отрисован корректно 1`] = `
<div
  className="page-body"
>
  <div
    className=""
  >
    <div
      className="menu heading-page"
    >
      <div
        className="heading-page-title"
      >
        История действий
      </div>
      <div
        className="heading-page-buttons"
      >
        <div
          className="button button__default"
          onClick={[Function]}
        >
          Свернуть историю
        </div>
      </div>
    </div>
    <div
      className="tableHistory"
    >
      <div
        className="tableHistory-row tableHistory-row__header"
      >
        <div
          className="tableHistory-col"
        >
          Пользователь
        </div>
        <div
          className="tableHistory-col"
        >
          Дата действия
        </div>
        <div
          className="tableHistory-col"
        >
          Действие
        </div>
      </div>
      <div
        className="tableHistory-row"
      >
        <div
          className="tableHistory-col"
        >
          Philip
        </div>
        <div
          className="tableHistory-col"
        >
          20.10.2020 10:22
        </div>
        <div
          className="tableHistory-col"
        >
          Вход в сервис
        </div>
      </div>
    </div>
  </div>
</div>
`;
